README.md

```diff
@@ -17,7 +17,21 @@ Check out our [documentation](https://wikunia.github.io/Javis.jl/dev/) for tutor
 
 ## Gallery
 
-![](assets/showcase.gif)
+### Tutorials
+
+| Animation                  | Animation                          |
```

Comment:
Can we remove the unnecessary header?

Reply:
Like maybe a good old fashion html table without borders :wink:

Reply:
I can't remove the header fully. I don't really want to mix HTML with markdown as I really don't like working with HTML if I can help it.

Reply:
Hmm that leave it as it is (revert the last commit as I think that looks odd)